project/vendor.sh

```diff
@@ -55,6 +55,8 @@ clone git github.com/docker/libtrust 230dfd18c232
 
 clone git github.com/Sirupsen/logrus v0.6.0
 
+clone git github.com/mistifyio/go-zfs f24c6bc9603ab5ad
```

Comment:
Can we adjust this to use upstream's `v2.1.0` tag instead?  If that tag isn't recent enough, we should see if upstream's willing to tag a new one. :+1: